portfolio/src/main/webapp/leaveamessage.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>About Kofi Meighan !</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+  <body bgcolor ="aliceblue">  
+    <h1 align="center">Kofi Meighan</h1>
+
+    <!--Personal info links at the top left of the screen-->
+    <ul class="info" align="left"><b>
+      <li><a href="https://github.com/kofimeighan">GitHub</a></li>
+      <li><a href="https://www.linkedin.com/in/kofimeighan/">LinkedIn</a></li>
+      <li>(561) 613-1936</li>
+      <li><a href="mailto:kofimeighan@google.com">
+        kofimeighan@google.com</a></li>
+      </b>
+    </ul>
+
+    <!--Navigation bar at the top of the screen-->
+    <ul class="nav">
+      <li><a href="index.html">Home</a></li>
+      <li><a href="education.html">Education & Activities</a></li>
+      <li><a href="pictures.html">Film Pictures</a></li>
+      <li><a href="mixes.html">Music Mixes</a></li>
+      <li><a href="leaveamessage.html">Leave me a message!</a></li>
+    </ul>
+    
+    	<body align="center" onload="checkLoginStatus(); getLimitedMessage();">
```

Comment:
With just two functions this is probably fine but it won't scale as you add many functions.

Prefer something like onload="initializePage()" or something like that and call checkLoginStatus() and getLimitedMessage() within that.

Reply:
Moved both functions into an onLoad() function